client/my-sites/plugins/plugin-details-CTA/index.jsx

```diff
@@ -226,7 +226,7 @@ const CTAButton = ( {
 				{
 					// eslint-disable-next-line no-nested-ternary
 					isMarketplaceProduct
-						? translate( 'Pay and install' )
+						? translate( 'Purchase and activate' )
 						: shouldUpgrade
 						? translate( 'Upgrade and install' )
```

Comment:
Should we also update `Upgrade and install` to `Upgrade and activate` since the plugin will get activated in all cases?

Reply:
I think it is a valid change to keep label uniformity.